x/profiles/keeper/grpc_query.go

```diff
@@ -129,11 +127,8 @@ func (k Keeper) ChainLinkOwners(ctx context.Context, request *types.QueryChainLi
 	store := sdkCtx.KVStore(k.storeKey)
 
 	ownersPrefix := types.ChainLinkChainPrefix
-	switch {
-	case request.ChainName != "" && request.Target != "":
+	if request.ChainName != "" {
 		ownersPrefix = types.ChainLinkChainAddressKey(request.ChainName, request.Target)
```

Comment:
Similar as above, can be less one case since `types.ChainLinkChainAddressKey("chain_name", "")` is equal to `types.ChainLinkChainKey("chain_name")`